thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobContext.java

```diff
@@ -7,10 +7,16 @@
 
 public class DetectionJobContext extends JobContext {
 
+  public enum DetectionJobType {
+    BACKFILL, OFFLINE
```

Comment:
should we have a DEFAULT DetectionJobType too, instead of making null for everything else